common/gyro-dashboard.cpp

```diff
@@ -0,0 +1,54 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#include "motion-dashboard.h"
+
+using namespace rs2;
+using namespace rsutils::string;
+
+gyro_dashboard::gyro_dashboard( std::string name )
+    : motion_dashboard( name )
+{
+}
+
+void gyro_dashboard::process_frame( rs2::frame f )
+{
+    write_shared_data(
+        [&]()
+        {
+            if( f && f.is< rs2::motion_frame >()
+                && ( f.as< rs2::motion_frame >() ).get_profile().stream_type() == RS2_STREAM_GYRO )
+            {
+                double ts = glfwGetTime();
+                auto it = frame_to_time.find( static_cast<float>( f.get_profile().unique_id() ) );
```

Comment:
If frame id is `int` why is the map key `float`?

Reply:
float removed